object/service.proto

```diff
@@ -262,7 +262,17 @@ service ObjectService {
   rpc Replicate(ReplicateRequest) returns (ReplicateResponse);
 }
 
-// GET object request
+// GET object request.
+//
+// The query for a parent object's EC part locally stored on the server is
```

Comment:
But it can be retrieved via part OID as well, right?

Reply:
yes, w/ regular GET request